src/NzbDrone.Core/MediaFiles/EpisodeImport/DetectSample.cs

```diff
@@ -67,7 +67,7 @@ public DetectSampleResult IsSample(Series series, string path, bool isSpecial)
                 return DetectSampleResult.Sample;
             }
 
-            _logger.Debug("Runtime is over 90 seconds");
+            _logger.Debug("[{0}] is not a sample. Runtime {1} seconds is more than minimum of {2} seconds", path, runTime, minimumRuntime);
```

Comment:
Using similar verbiage to is a sample text

```suggestion
            _logger.Debug("[{0}] does not appear a sample. Runtime {1} seconds is more than minimum of {2} seconds", path, runTime, minimumRuntime);
```